python/src/robyn/modeling/ridge_model_builder.py

```diff
@@ -731,5 +750,6 @@ def _lambda_seq(
         seq_len: int = 100,
         lambda_min_ratio: float = 0.0001,
     ) -> np.ndarray:
+        # R calculates lambda_max differently, need to match their scaling
```

Comment:
Nit: These LLM generated doc doesnt seem apt here.

Reply:
the next PR will address editing all docstrings in bulk